.github/workflows/pr-reminder.yml

```
@@ -0,0 +1,217 @@
+name: TapTap PR Reminder
+
+on:
+  pull_request:
+    types: [opened, reopened]
+  issue_comment:
+    types: [created]
+  schedule:
+    # 1분마다 체크 (중복 방지 label로 실제 알림은 24시간 이후)
+    - cron: "* * * * *"
```

actionlint (1.7.10)

[error] scheduled job runs too frequently. it runs once per 60 seconds. the shortest interval is once every 5 minutes

(events)